memory.go

```diff
@@ -243,6 +243,27 @@ func (mb *MemoryBackend) Select(slct *SelectStatement) (*Results, error) {
 		t.rows = [][]MemoryCell{{}}
 	}
 
+	// handle SELECT *
+	if len(*slct.item) == 1 && (*slct.item)[0].asterisk {
```

Comment:
I don't think this should or needs to be handled as a special case. I assume the behavior of using `*` is to display all columns that haven't already been a part of the select column statement. So I think we'd want some logic to look through all columns referenced and only add the rest.

Whatever the case, I don't think it's actually a special case?

Reply:
I just checked and in postgres it adds all columns, including ones that are already a part of the SELECT statement